test/test_mahalanobis_mixin.py

```diff
@@ -138,9 +139,6 @@ def test_embed_dim(estimator, build_dataset):
   assert str(raised_error.value) == err_msg
   # we test that the shape is also OK when doing dimensionality reduction
   if type(model).__name__ in {'LFDA', 'MLKR', 'NCA', 'RCA'}:
-    # TODO:
```

Comment:
This will be changed (for good) in #193 (see https://github.com/metric-learn/metric-learn/pull/193/files#diff-2d2fa8dfd3b4bf0d149166d5e3efa526)